pom.xml

```diff
@@ -13,6 +13,13 @@
     <name>Jenkins batch task plugin</name>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Batch+Task+Plugin</url>
 
+    <properties>
+        <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <powermock.version>1.6.1</powermock.version>
+        <findbugs-maven-plugin.version>3.0.2</findbugs-maven-plugin.version>
```

Comment:
This can be removed (together with the plugin definition) since this is defined in the parent pom.